app/locales/sk/locationTracking.json

```diff
@@ -0,0 +1,13 @@
+{
+    "start_logging":"ZAČAŤ ZAZNAMENÁVAŤ",
+    "stop_logging":"PRESTAŤ ZAZNAMENÁVAŤ",
+    "logging_message":"Práve sa súkromne zaznamenáva tvoja poloha každých päť minút. Informácie o tvojej polohe NEOPUSTIA tvoj telefón.",
```

Comment:
Práve sa každých päť minút súkromne zaznamenáva tvoja poloha. - sounds better

Reply:
"vaša" is more formal than "tvoja"

Reply:
"Vaša / Tvoja poloha sa teraz zaznamenáva každých päť minút." is more natural

Reply:
Consider changing "o tvojej" to "o vašej" and "tvoj" to "váš"